CHANGELOG.md

```diff
@@ -11,6 +11,114 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.9.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.9.0.
+This release brings new features such as:
+- Supporting half precision (IEEE FP16). The type `gko::half` can now be selected in most instances as the value type
+  of a matrix, solver, preconditioner, etc. If the selected backend supports FP16 as a native type, the native type is
+  used within the kernels, otherwise an overhead might occur. The new behavior is enabled by default, but it can be
+  turned off during configuration.
+- Providing new implementations of the ILU and IC factorization for CUDA, HIP, OpenMP, and Reference backends. These are
+  available in addition to the existing implementations based on the vendor libraries cuSPARSE and hipSPARSE.
+- Adding new (S)SOR and Gauß-Seidel preconditioners.
+- Simplifying the distributed matrix assembly by gathering local rows from neighboring processes.
+
+And more!
+
+If you face an issue, please first check our [known issues page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues) and the [open issues list](https://github.com/ginkgo-project/ginkgo/issues) and if you do not
+find a solution, feel free to [open a new issue](https://github.com/ginkgo-project/ginkgo/issues/new/choose) or ask a question using the [github discussions](https://github.com/ginkgo-project/ginkgo/discussions).
+
+Supported systems and requirements:
++ For all platforms, CMake 3.16+
++ C++17 compliant compiler
++ Linux and macOS
+  + GCC: 7.0+
+  + clang: 5.0+
+  + Intel compiler: 2019+
+  + Apple Clang: 15.0 is tested. Earlier versions might also work.
+  + NVHPC: 22.7+
+  + Cray Compiler: 14.0.1+
+  + CUDA module: CMake 3.18+, and CUDA 11.0+ or NVHPC 22.7+
+  + HIP module: CMake 3.21+, and ROCm 4.5+
+  + DPC++ module: Intel oneAPI 2023.1+ with oneMKL and oneDPL. Set the CXX compiler to `dpcpp` or `icpx`.
+  + MPI: standard version 3.1+, ideally GPU Aware, for best performance
++ Windows
+  + MinGW: GCC 7.0+
+  + Microsoft Visual Studio: VS 2019+
+  + CUDA module: CUDA 11.0+, Microsoft Visual Studio
+  + OpenMP module: MinGW.
+
+### Version support changes
++ Ginkgo now requires a compiler with C++ 17 support [#1603](https://github.com/ginkgo-project/ginkgo/pull/1603)
+
+### Deprecations
+
++ The `Executor::run` overload taking in multiple functions without a name as first parameter has been deprecated [#1667](https://github.com/ginkgo-project/ginkgo/pull/1667)
+
+### Summary of previous deprecations
```

Comment:
This and the deprecations header above are at the same heading level. Should this one be a part of the one above ?

Reply:
That's a good idea. I will make it a sub-heading.